### ISSUE TITLE
Add timer so that the same problem alert can't pop up too soon after you close it

### ISSUE THREAD

**Issue by nwinter** [codecombat/codecombat#2929]
A suggestion from [the forum](http://discourse.codecombat.com/t/first-impressions-and-interface-and-ux-suggestions/4652):

> **Tips coming up waaay too often**
> - my sis was well aware that she mis-spelled the enemy's name and was in the middle of correcting it, but no matter how often she closed the help notification, it kept coming up after every keystroke
> - soluton: use a timer to see if the user has stopped writing and only activate these pop-ups then

This would touch [ProblemAlertView.coffee](https://github.com/codecombat/codecombat/blob/master/app/views/play/level/tome/ProblemAlertView.coffee) and [SpellView.coffee](https://github.com/codecombat/codecombat/blob/master/app/views/play/level/tome/SpellView.coffee#L596-L597).

**Comment by stevenkaza** [comment]
Is this still an issue, can't seem to reproduce it.

**Comment by nwinter** [comment]
Yeah, especially if the mouse cursor moves back over the line that has the problem right after closing the error message.
